src/shared/components/FormComponents/InputController/Input/Input.const.ts

```diff
@@ -0,0 +1 @@
+export const INPUT_DEBOUNCE_VALUE = 500;
```

Comment:
maybe it's better to keep this constants in shared/constants or so

Reply:
done